test/check-machines-create

```diff
@@ -789,8 +790,10 @@ vnc_password= "{vnc_passwd}"
                 b.wait_in_text(".pf-c-modal-box .pf-c-modal-box__header .pf-c-modal-box__title", "Create new virtual machine")
 
             if self.os_name is not None:
+                # remove os codename from os name, e.g.: 'bullseye' in 'Debian 11 (bullseye)'
+                query_result = re.sub(r" \(.*\)", "", self.os_name)
+                query_result = f'{query_result}'
```

Comment:
This seems quite unrelated? Please add a "why" to the commit message. In particular, can I still search for the code names and get a result? The [release names](https://www.debian.org/releases/) are quite popular, and easy to google for.

Reply:
added to the commit message.
```
In the UI, we try to create a full OS name by combining a distro,
version and a release name, e.g. Red Hat Enterprise Linux 9.0 (Plow).
However osinfo-query output doesn't contain release name, therefore
we need ommit it in our tests when matching osinfo-query output with
an OS list shown in the UI.
```
So basically, in the UI, we show OS with a string like "Red Hat Enterprise Linux 9.0 (Plow)", and then check if it is also present in the `$osinfo-query os` command output. But `osinfo-query os` only contains string "Red Hat Enterprise Linux 9.0", without the release name. So here I just get rid of the release name, and then try to match the OS name from the UI with an `osinfo-query os` command output.